shared/constants/multichain/networks.ts

```diff
@@ -99,8 +121,15 @@ export const MULTICHAIN_TOKEN_IMAGE_MAP: Record<CaipChainId, string> = {
   [MultichainNetworks.BITCOIN]: BITCOIN_TOKEN_IMAGE_URL,
   [MultichainNetworks.BITCOIN_TESTNET]: BITCOIN_TESTNET_TOKEN_IMAGE_URL,
   [MultichainNetworks.SOLANA]: SOLANA_TOKEN_IMAGE_URL,
+  [MultichainNetworks.SOLANA_DEVNET]: SOLANA_DEVNET_IMAGE_URL,
+  [MultichainNetworks.SOLANA_TESTNET]: SOLANA_TESTNET_IMAGE_URL,
 } as const;
 
+/**
+ * @deprecated MULTICHAIN_PROVIDER_CONFIGS is deprecated and will be removed in the future.
+ * Use the data from @metamask/multichain-network-controller.
+ * Useful selectors in selectors/multichain/networks.ts
```

Comment:
Very nit: (I case you pickup other suggestions)

```suggestion
 * Useful selectors in selectors/multichain/networks.ts.
```

Reply:
https://github.com/MetaMask/metamask-extension/pull/31702/commits/951c0efd729e180d881e0c51ffdc75f9f75a2960